source/openapi3/split.yaml

```
@@ -2536,7 +2294,7 @@ paths:
       tags:
         - Contacts
       summary: Refresh contact bank connection
-      description: Request the bank connection for a contact to refresh available funds. This is intended for use in conjunction with the `precheck_funds` option, see [Payment Request - Precheck Funds](/#precheck-funds-lifecycle)
+      description: Refresh one of your contact's bank connections to get the latest available balance.
```

Are we capitalising actors?
```suggestion
      description: Refresh one of your Contact's bank connections to get the latest available balance.
```